Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contests-filters/ContestFilters.tsx

```diff
@@ -54,83 +47,24 @@ const ContestFilters = ({ onFilterClick }: IContestFiltersProps) => {
         [ possibleFilters, onFilterClick ],
     );
 
-    const getRenderFilterItemFunc = useCallback(
-        (type: FilterType) => ({ id, name }: IFilter) => {
-            // TODO: investigate why filters change ids
-            //  and use id instead of name for checking if filter is selected
-            const filterIsSelected = filters.some((f) => f.name === name);
-            const buttonType = filterIsSelected
-                ? ButtonType.primary
-                : ButtonType.secondary;
-
-            const btnClassName = type === FilterType.Status
-                ? styles.btnSelectFilter
-                : '';
-
-            return (
-                <Button
-                  type={buttonType}
-                  onClick={() => handleFilterClick(id)}
-                  className={btnClassName}
-                  text={name}
-                  size={ButtonSize.small}
-                />
-            );
-        },
-        [ handleFilterClick, filters ],
-    );
-
-    const toggleFiltersExpanded = useCallback(
-        (isExpanded) => setExpanded(isExpanded),
-        [],
-    );
-
-    const renderExpandButton = useCallback(
-        (allFilters: IFilter[]) => {
-            const maxFiltersToDisplayCount = 3;
-
-            return allFilters.length > maxFiltersToDisplayCount
-                ? <ExpandButton onExpandChanged={toggleFiltersExpanded} />
-                : null;
-        },
-        [ toggleFiltersExpanded ],
-    );
-
     const renderFilter = useCallback(
         (fg: IFiltersGroup) => {
             const { type, filters: groupFilters } = fg;
-            const className = concatClassNames(
-                styles.listFilters,
-                expanded
-                    ? styles.expanded
-                    : '',
-            );
 
             const listOrientation = type === FilterType.Status
                 ? Orientation.horizontal
                 : Orientation.vertical;
 
             return (
-                <div className={styles.filterTypeContainer}>
-                    <Heading
-                      type={HeadingType.small}
-                      className={styles.heading}
-                    >
-                        {type}
-                    </Heading>
-                    <List
-                      values={groupFilters}
-                      itemFunc={getRenderFilterItemFunc(type)}
-                      orientation={listOrientation}
-                      className={className}
-                      itemClassName={styles.listFilterItem}
-                      fullWidth
-                    />
-                    {renderExpandButton(groupFilters)}
-                </div>
+                <ContestFilterType
```

Comment:
Can be ContestFilter instead of ContestFilterType in order to be clear that the component is 'ContestFilter'